boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -348,9 +353,21 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
         TRACE("Type: 0x%lx\n", PcBiosMemoryMap[PcBiosMapCount].Type);
         TRACE("ExtendedAttributesAsULONG: 0x%08lx\n", PcBiosMemoryMap[PcBiosMapCount].ExtendedAttributesAsULONG);
 
+        if (PcBiosMemoryMap[PcBiosMapCount].ExtendedAttributes.ErrorLog == 1)
+        {
+            FIXME("EA.ErrorLog = 1. Please report this to CORE-14150. "
```

Comment:
> So the point of this is to add a debug print that nobody will ever see?

Why would this `FIXME()` never be seen, precisely?
Hack-test: It works fine on VPC 2004, though the latter does not return EAs...